src/api/forms/service.js

```diff
@@ -124,19 +134,14 @@ export function getFormDefinition(formId, state = 'draft') {
 export async function updateDraftFormDefinition(formId, definition, author) {
   const existingForm = await getForm(formId)
 
-  if (!existingForm) {
-    throw Boom.notFound(
-      `Form ${formId} does not exist, so the definition cannot be updated.`
-    )
-  }
-
   // Throw if there's no current draft state
   if (!existingForm.draft) {
     throw Boom.badRequest(`No 'draft' state found for form metadata ${formId}`)
   }
 
   // Update the form definition
   await draftFormDefinition.create(formId, definition)
+  logger.info(`Draft form definition updated for form ID ${formId}`)
```

Comment:
Should we log this message from within `.create()` instead?

Reply:
Done